circle.yml

```diff
@@ -1,29 +0,0 @@
-machine:
-  services:
-    - docker
-dependencies:
-  override:
-    - cp Dockerfile Dockerfile.run
-    - cp Dockerfile.build Dockerfile
-    - docker build -t build-img .
-    - docker create --name build-cont build-img
-    - docker cp build-cont:/usr/local/gaia/target/mgs.war ./target/mgs.war
-    - cp Dockerfile.run Dockerfile
-    - docker build -t gaiaadm/mgs .
-test:
```

Comment:
Do you plan to do end-to-end test? checking that the message reached to RabbitMQ with the right format?